e2e-win/download_long_path.Tests.ps1

```
@@ -0,0 +1,92 @@
+    }
+
+    AfterAll {
+        Set-Location $script:OriginalDir
+        foreach ($v in $script:Saved.Keys) {
+            if ($null -ne $script:Saved[$v]) { Set-Item "Env:\$v" $script:Saved[$v] }
+            else { Remove-Item "Env:\$v" -ErrorAction SilentlyContinue }
+        }
+        # A tree past MAX_PATH cannot be removed through the ordinary API, and Pester hands the
+        # drive back by walking it.
+        if (Test-Path -LiteralPath $script:LongData) {
+            try { [System.IO.Directory]::Delete("\\?\$($script:Root)", $true) } catch { }
```

PSScriptAnalyzer (1.25.0)

[warning] Empty catch block is used. Please use Write-Error or throw statements in catch blocks.

(PSAvoidUsingEmptyCatchBlock)